samples/queryShortMode.js

```diff
@@ -17,12 +17,13 @@
 function main() {
   // [START bigquery_query_shortmode]
```

Comment:
can you update this?  bigquery_query_job_optional

Reply:
done